src/scripts/modules/media/body/content-baked.less

```diff
@@ -31,6 +31,16 @@ ul, ol {
   padding-left: 1rem;
 }
 
+.stepwise {
+  list-style-type: none;
+  counter-reset: stepwiseCounter;
+  padding-left: 0;
+  > li:before {
+      content: "Step " counter(stepwiseCounter) ". ";
+      counter-increment: stepwiseCounter;
+  }
+}
```

Comment:
I think you can delete margin for list items if you think it shouldn't be here:
```suggestion
 > li {
       margin-bottom: 0;
   }
}
```

Reply:
Yes, what about list without `class="stepwise"` like here?
![image](https://user-images.githubusercontent.com/20907906/48196744-ada71880-e353-11e8-9c06-af9a23002034.png)

Reply:
I see some options:
1. Remove global margin for `<li>` and set margin depends of parent's class.
2. Leave global margin as it is and set custom margin for `<li>` depends of parent's class.
3. Lower global margin, ex: from `1em` to `0.8em` so change is not visible that much.